tox.ini

```diff
@@ -51,6 +51,14 @@ basepython = python3
 deps = {[lint]deps}
 commands = {[lint]commands}
 
+[testenv:format]
+deps =
+    black==18.5b0
+commands =
+    bash -c "find src tasks tests -not -path 'src/pip/_vendor/*' -name '*.py' -type f -print0 | xargs -0 black"
```

Comment:
I hadn't looked at this code until now. Can't we just do `black src/pip/_internal tests tasks`?

Reply:
Oh, that missed out on `pip.__init__` and `pip.__main__`. At which point, it starts getting unwieldy to use.

Sorry for the noise.